package.json

```diff
@@ -39,33 +39,32 @@
     }
   ],
   "dependencies": {
-    "atom-babel6-transpiler": "1.2.0",
-    "babel-generator": "6.26.1",
+    "@atom/babel7-transpiler": "1.0.0-1",
+    "@babel/generator": "7.3.4",
+    "@babel/plugin-proposal-class-properties": "7.3.4",
+    "@babel/preset-env": "7.3.4",
+    "@babel/preset-react": "7.0.0",
     "babel-plugin-chai-assert-async": "0.1.0",
-    "babel-plugin-relay": "1.7.0",
-    "babel-plugin-transform-class-properties": "6.24.1",
-    "babel-plugin-transform-es2015-modules-commonjs": "6.26.2",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-preset-react": "6.24.1",
+    "babel-plugin-relay": "3.0.0",
     "bintrees": "1.0.2",
     "bytes": "^3.0.0",
     "classnames": "2.2.6",
     "compare-sets": "1.0.1",
     "dugite": "^1.81.0",
     "event-kit": "2.5.3",
     "fs-extra": "4.0.3",
-    "graphql": "0.13.2",
+    "graphql": "14.1.1",
```

Comment:
This is...quite the jump

Reply:
If it helps, it looks like graphql jumped from the 0.13 line to the 14.x line directly.

Reply:
Oh haha, that makes a lot more sense 😄